hw/ip/otbn/data/base-insns.yml

```
@@ -393,6 +395,8 @@
     Reads the old value of the CSR, and writes it to the GPR `grd`.
     Writes the initial value in `grs1` to the CSR `csr`.
     If `grd == x0` the instruction does not read the CSR or cause any read-related side-effects.
+
+    If `csr` isn't the index of a valid CSR, this results in an error (setting bit `illegal_insn` in `ERR_BITS`).
```

Since we don't have write-only CSRs the following addition should do:

```suggestion
    If `csr` isn't the index of a valid CSR, or if the CSR is not writable, this results in an error (setting bit `illegal_insn` in `ERR_BITS`).
```